crates/ruff_python_formatter/tests/snapshots/black_compatibility@simple_cases__fmtonoff.py.snap

```diff
@@ -198,192 +198,47 @@ d={'a':1,
+@@ -63,7 +63,7 @@
  
  something = {
      # fmt: off
 -    key: 'value',
-+    key: "value",
++    key: 'value'
  }
  
  
- def subscriptlist():
-     atom[
-         # fmt: off
--        'some big and',
--        'complex subscript',
-+        "some big and",
-+        "complex subscript",
-         # fmt: on
-         goes + here,
-         andhere,
-@@ -80,38 +94,36 @@
- 
- def import_as_names():
+@@ -122,8 +122,10 @@
+     """
      # fmt: off
--    from hello import a,        b
--    'unformatted'
-+    from hello import a, b
-+
-+    "unformatted"
-     # fmt: on
  
+-    # hey, that won't work
  
- def testlist_star_expr():
-     # fmt: off
--    a , b = *hello
--    'unformatted'
-+    a, b = *hello
-+    "unformatted"
++        #hey, that won't work
++
++
      # fmt: on
+     pass
  
- 
- def yield_expr():
-     # fmt: off
-     yield hello
--    'unformatted'
-+    "unformatted"
-     # fmt: on
-     "formatted"
-     # fmt: off
--    ( yield hello )
--    'unformatted'
-+    (yield hello)
-+    "unformatted"
+@@ -136,9 +138,9 @@
+     and_=indeed . it  is  not  formatted
+     because . the . handling . inside . generate_ignored_nodes()
+     now . considers . multiple . fmt . directives . within . one . prefix
++        # ...but comments still get reformatted even though they should not be
      # fmt: on
- 
- 
- def example(session):
      # fmt: off
--    result = session\
--        .query(models.Customer.id)\
--        .filter(models.Customer.account_id == account_id,
--                models.Customer.email == email_address)\
--        .order_by(models.Customer.id.asc())\
--        .all()
-+    result = session.query(models.Customer.id).filter(
-+        models.Customer.account_id == account_id, models.Customer.email == email_address
-+    ).order_by(models.Customer.id.asc()).all()
+-    # ...but comments still get reformatted even though they should not be
      # fmt: on
  
  
-@@ -132,10 +144,10 @@
-     """Another known limitation."""
-     # fmt: on
-     # fmt: off
--    this=should.not_be.formatted()
--    and_=indeed . it  is  not  formatted
--    because . the . handling . inside . generate_ignored_nodes()
--    now . considers . multiple . fmt . directives . within . one . prefix
-+    this = should.not_be.formatted()
-+    and_ = indeed.it is not formatted
-+    because.the.handling.inside.generate_ignored_nodes()
-+    now.considers.multiple.fmt.directives.within.one.prefix
-     # fmt: on
-     # fmt: off
-     # ...but comments still get reformatted even though they should not be
-@@ -153,9 +165,7 @@
-             )
-         )
-         # fmt: off
--        a = (
--            unnecessary_bracket()
--        )
-+        a = unnecessary_bracket()
-         # fmt: on
-     _type_comment_re = re.compile(
-         r"""
-@@ -178,7 +188,7 @@
+@@ -178,7 +180,7 @@
          $
          """,
          # fmt: off
 -        re.MULTILINE|re.VERBOSE
-+        re.MULTILINE | re.VERBOSE,
++        re.MULTILINE|re.VERBOSE,
```

Comment:
TODO: double check why this trailing comma is added and what happens if the argument already had a trailing comma